### PR TITLE
fixed a few errors in the INSTALL.md

### PR DESCRIPTION
**Pull request by GodloveD**
**Description of the Pull Request (PR):**

The old version of `INSTALL.md` was using `apt` to install golang.  This was getting an older version of golang that would bug out when trying to install `dep`.  There was also a dependency that was not accounted for. 

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [x] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- [ ] I have tested this PR locally with a `make test`
- [x] This PR is NOT against the project's master branch
- [x] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [x] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -8,16 +8,33 @@ You must first install development and libraries to your host. Assuming Ubuntu
...
+$ sudo tar -C /usr/local -xzf go$VERSION.$OS-$ARCH.tar.gz
+```
+
+Finally, set up your environment for go
+
 ```
-$ sudo apt-get update && sudo apt-get install -y golang
 $ echo 'export GOPATH=$HOME/go' >> ~/.bashrc
-$ echo 'export PATH=${PATH}:${GOPATH}/bin' >> ~/.bashrc
+$ echo 'export PATH=/usr/local/go/bin:${PATH}:${GOPATH}/bin' >> ~/.bashrc
```
Comment (jmstover) [inline INSTALL.md]: Adding in `/usr/local/go` ... would you set:
```
export GOROOT=/usr/local/go
```

In there as well?

Comment (GodloveD) [reply INSTALL.md]: I'm no expert, but I think GOROOT is only necessary if you install in a custom location?
https://golang.org/doc/install

Comment (ArangoGutierrez) [reply INSTALL.md]: `GOROOT` must be set only when installing to a custom location.
if `GOROOT` == `/usr/local/go` , it can be empty.

**Review by ArangoGutierrez — DISMISSED** [review]

File: INSTALL.md (hunk 1 above)
Comment (jmstover) [reply INSTALL.md]: Ahh... when using the *binary* distribution.  From the golang site:
_"The Go binary distributions  assume they will be installed in /usr/local/go"_

Comment (GodloveD) [reply INSTALL.md]: OK.  So, good to go then?

Comment (ArangoGutierrez) [reply INSTALL.md]: Hit that big green button!!

File: INSTALL.md [hunk 2]
```diff
@@ -8,16 +8,33 @@ You must first install development and libraries to your host. Assuming Ubuntu
...
+
+```
+$ sudo tar -C /usr/local -xzf go$VERSION.$OS-$ARCH.tar.gz
+```
+
+Finally, set up your environment for go
+
 ```
-$ sudo apt-get update && sudo apt-get install -y golang
 $ echo 'export GOPATH=$HOME/go' >> ~/.bashrc
```
Comment (bauerm97) [inline INSTALL.md]: Should this be `${HOME} ` to align with the env var formatting in line 37?

**Review by bauerm97 — CHANGES_REQUESTED** [review]
If you wanted to change this formatting, that would be neat. Sorry for being super nitpicky lol!

File: INSTALL.md [hunk 3]
```diff
@@ -8,16 +8,33 @@ You must first install development and libraries to your host. Assuming Ubuntu
 (apply similar to RHEL derivatives):
 
 ```
-$ sudo apt-get update && sudo apt-get install -y build-essential libssl-dev uuid-dev
+$ sudo apt-get update && sudo apt-get install -y build-essential libssl-dev uuid-dev libgpgme-dev
```
Comment (yhcote) [inline INSTALL.md]: why do we need libgpgme again ?

Comment (bauerm97) [reply INSTALL.md]: I don't remember why, I think it was from some docker image building thing?

Anyway, the proper package to install on Ubuntu is `libgpgme11-dev`

Comment (GodloveD) [reply INSTALL.md]: So should I change this?  Or is it good the way it is?

**Review by bauerm97 — APPROVED** [review]

**Pull request merged**